kujira/assetlist.json

```diff
@@ -233,6 +233,65 @@
       "logo_URIs": {
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usdt.svg"
       }
+    },
+    {
+      "description": "whLOCAL on Kujira",
+      "denom_units": [
+        {
+          "denom": "ibc/DE92DB84EA7E8CB5EC76BC395CF634D6483B67414AD28BEF3F71EBFD6A780D36",
+          "exponent": 0
+        },
+        {
+          "denom": "whLOCAL",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/DE92DB84EA7E8CB5EC76BC395CF634D6483B67414AD28BEF3F71EBFD6A780D36",
+      "name": "whLOCAL",
+      "display": "whLOCAL",
+      "symbol": "whLOCAL",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "terra2",
+            "base_denom": "cw20:terra1xzkel96e5e8vfmqw7valzdzzv9hqasfyslclvnmvxdejvpda3xwsskxzus"
+            "channel_id": "channel-XX"
```

Comment:
I wasn't able to find the remote (the terra2 side) channel id, and this IBC channel hasn't been registered--please register it to _IBC/kujira-terra2.json

Reply:
@JeremyParish69 I have made some changes to the code to what I believe is the right channel to use base on my research on https://chainsco.pe/terra2/ibc/connections/connection-13 and https://finder.kujira.network/kaiyo-1/token/ibc%2FDE92DB84EA7E8CB5EC76BC395CF634D6483B67414AD28BEF3F71EBFD6A780D36

Happy to amend anything base on your feedback

Reply:
Sure, we can use that info, but the IBC Connection needs to first be defined in '_IBC/kujira-terra2.json', which it currently is not.

Reply:
@JeremyParish69 I'm gonna need some help with this one as I couldn't see anything specific on the schema for that file, I'm sorry very new to IBC, could you expand a bit on what's the connection you are expecting to see in `_IBC/kujira-terra2.json`, thanks

Reply:
Sure, if you look at the Juno-osmosis ibc connection file, for example, we can see that they've defined an ibc connection with a native channel and also a [cw20-specific channel](https://github.com/cosmos/chain-registry/blob/1a9d79ed3affee1bfaf6bad1696dbaae3d36cbbc/_IBC/juno-osmosis.json#L31C2-L46C10). We need a cw20 specific channel to be defined in the kujira-terra2.json file.

Reply:
As another example, it looks like someone else also registered a different wasm channel from terra2 to kujira.
But it looks like this token isn't using the same channel. It would be best to use the same channel for all cw20s, but we are able to register mulitple.
https://github.com/cosmos/chain-registry/blob/b539099ada02a25033be7fcdb868f5321296fd26/_IBC/kujira-terra2.json#L37

Reply:
So wait so does that means that they already did it? so we don't need to change the kujira-terra2.json file? because my changes looked exactly the same that those

Reply:
I did implemented some changes as you suggested @JeremyParish69 but still a bit confuse as it seems like this changes https://github.com/cosmos/chain-registry/blob/b539099ada02a25033be7fcdb868f5321296fd26/_IBC/kujira-terra2.json#L37 are the changes I was aiming for however can't see how is that block related to our specific token

Reply:
Thinking about it shouldn't this a block like this what we need? for the whLOCAL token? 🤔

```
{
      "chain_1": {
        "channel_id": "channel-10",
        "port_id": "wasm.terra1e0mrzy8077druuu42vs0hu7ugguade0cj65dgtauyaw4gsl4kv0qtdf2au"
      },
      "chain_2": {
        "channel_id": "channel-43",
        "port_id": "transfer"
      },
      "ordering": "unordered",
      "version": "ics20-1",
      "tags": {
        "status": "live",
        "preferred": true
      }
    }
```

Reply:
So, currently I see a standard ibc connection between kujira and terra2 (5/10), and I also see an additional 28/36 wasm ibc connection there as well. But there is still no 10/43 wasm ibc connection registered. You might be able to use the same (28/36) ibc connection that's already registered, but that would result in a different ibc denomination for $LOCAL, and any tokens already transferred via 10/43 would be invalid.

Reply:
Also I'm skeptical that both the 10/43 and the 28/36 would use the same wasm port, but it may indeed be correct...

Reply:
But yes, to have ibc/DE92DB84EA7E8CB5EC76BC395CF634D6483B67414AD28BEF3F71EBFD6A780D36 be considered valid, you'd need a block like the one you posted 3 comments above to register the channel-43 connection.